src/cmd/mod.rs

```diff
@@ -1,5 +1,4 @@
 pub mod agent;
 pub mod balancer;
 
-#[cfg(feature = "ratatui_dashboard")]
```

Comment:
So far the PR looks great. Can you make sure that `ratatui_dashboard` is behind a feature flag and Paddler works and compiles without `ratatui_dashboard` feature flag enabled?